<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Battle Map</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="GraphicsView_BattleMap" name="GraphicsView_BattleMapMasterScreen">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1880</width>
      <height>1060</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="interactive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Label_ScaleFactor">
    <property name="geometry">
     <rect>
      <x>1810</x>
      <y>1020</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Label_WindRose">
    <property name="geometry">
     <rect>
      <x>1670</x>
      <y>10</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="MenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="Menu_File">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="Action_NewBattleMap"/>
    <addaction name="Action_OpenBattleMap"/>
    <addaction name="separator"/>
    <addaction name="Action_Quit"/>
   </widget>
   <widget class="QMenu" name="Menu_Edit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="Menu_View">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="SubMenu_WindRoseOrientation">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Wind Rose Orientation</string>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icons/icons/windrose.png</normaloff>:/icons/icons/windrose.png</iconset>
     </property>
     <addaction name="Action_WindRoseOrientationNorth"/>
     <addaction name="Action_WindRoseOrientationEast"/>
     <addaction name="Action_WindRoseOrientationSouth"/>
     <addaction name="Action_WindRoseOrientationWest"/>
    </widget>
    <addaction name="Action_UpdatePlayerScreen"/>
    <addaction name="separator"/>
    <addaction name="SubMenu_WindRoseOrientation"/>
    <addaction name="separator"/>
    <addaction name="Mode_Select"/>
    <addaction name="Mode_CoverBattleMap"/>
    <addaction name="Mode_UncoverBattleMap"/>
   </widget>
   <widget class="QMenu" name="Menu_Settings">
    <property name="title">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="QMenu" name="Menu_Help">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="Menu_File"/>
   <addaction name="Menu_Edit"/>
   <addaction name="Menu_View"/>
   <addaction name="Menu_Settings"/>
   <addaction name="Menu_Help"/>
  </widget>
  <widget class="QToolBar" name="ToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="Action_UpdatePlayerScreen"/>
   <addaction name="separator"/>
   <addaction name="Mode_Select"/>
   <addaction name="Mode_CoverBattleMap"/>
   <addaction name="Mode_UncoverBattleMap"/>
  </widget>
  <action name="Action_NewBattleMap">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/newbattlemap.png</normaloff>:/icons/icons/newbattlemap.png</iconset>
   </property>
   <property name="text">
    <string>New Battle Map</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="Action_Quit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/quit.png</normaloff>:/icons/icons/quit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="Action_OpenBattleMap">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/openbattlemap.png</normaloff>:/icons/icons/openbattlemap.png</iconset>
   </property>
   <property name="text">
    <string>Open Battle Map</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="Action_UpdatePlayerScreen">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/updateplayerscreen.png</normaloff>:/icons/icons/updateplayerscreen.png</iconset>
   </property>
   <property name="text">
    <string>Update Player Screen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="Mode_UncoverBattleMap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/uncoverbattlemap.png</normaloff>:/icons/icons/uncoverbattlemap.png</iconset>
   </property>
   <property name="text">
    <string>Uncover Battle Map</string>
   </property>
   <property name="toolTip">
    <string>Uncover Battle Map</string>
   </property>
  </action>
  <action name="Mode_CoverBattleMap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/coverbattlemap.png</normaloff>:/icons/icons/coverbattlemap.png</iconset>
   </property>
   <property name="text">
    <string>Cover Battle Map</string>
   </property>
   <property name="toolTip">
    <string>Cover Battle Map</string>
   </property>
  </action>
  <action name="Mode_Select">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/cursor.png</normaloff>:/icons/icons/cursor.png</iconset>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
   <property name="iconText">
    <string>Default Cursor</string>
   </property>
   <property name="toolTip">
    <string>Select</string>
   </property>
  </action>
  <action name="Action_UncoverBattleMap">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/uncoverbattlemap.png</normaloff>:/icons/icons/uncoverbattlemap.png</iconset>
   </property>
   <property name="text">
    <string>Uncover Battle Map</string>
   </property>
  </action>
  <action name="Action_CoverBattleMap">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/coverbattlemap.png</normaloff>:/icons/icons/coverbattlemap.png</iconset>
   </property>
   <property name="text">
    <string>Cover Battle Map</string>
   </property>
  </action>
  <action name="Action_WindRoseOrientationNorth">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>North</string>
   </property>
  </action>
  <action name="Action_WindRoseOrientationEast">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>East</string>
   </property>
  </action>
  <action name="Action_WindRoseOrientationSouth">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>South</string>
   </property>
  </action>
  <action name="Action_WindRoseOrientationWest">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>West</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsView_BattleMap</class>
   <extends>QGraphicsView</extends>
   <header location="global">graphicsview_battlemap.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>buttonClick()</slot>
 </slots>
</ui>
